externals/miniz/miniz.c

```diff
@@ -0,0 +1,7557 @@
+/**************************************************************************
```

Comment:
This has been added into the project instead of as a submodule. Intended? Same applies to header.

Reply:
miniz.c and h are generated from multiple files. So one is not supposed to compile directly from the git repository but from releases.

Reply:
Annoying for us :/ Oh well, nothing we can do about it, I guess.